{"uuid":"69496913-6def-496b-a7ac-a7b3ddeda37d","historyId":"89158d245f7fe5feb1feaff33b2ad47f","fullName":"testCases.PasswordRecoveryTest.RecoveryPopUpSupportLink_Test","labels":[{"name":"package","value":"testCases.PasswordRecoveryTest"},{"name":"testClass","value":"testCases.PasswordRecoveryTest"},{"name":"testMethod","value":"RecoveryPopUpSupportLink_Test"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"testCases"},{"name":"subSuite","value":"testCases.PasswordRecoveryTest"},{"name":"host","value":"DESKTOP-PO3T4V3"},{"name":"thread","value":"17352@DESKTOP-PO3T4V3.TestNG-tests-1(19)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"minor"}],"links":[],"name":"Validate on Password Recovery Pop Up Support link","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]","trace":"java.lang.AssertionError: did not expect [https://craftbet.com/forgot-password] but found [https://craftbet.com/forgot-password]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failEquals(Assert.java:1417)\r\n\tat org.testng.Assert.assertNotEqualsImpl(Assert.java:156)\r\n\tat org.testng.Assert.assertNotEquals(Assert.java:2033)\r\n\tat org.testng.Assert.assertNotEquals(Assert.java:2055)\r\n\tat testCases.PasswordRecoveryTest.RecoveryPopUpSupportLink_Test(PasswordRecoveryTest.java:84)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:806)\r\n\tat org.testng.TestRunner.run(TestRunner.java:601)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:433)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\n"},"stage":"finished","description":"Validate on Password Recovery Pop Up Support link","steps":[],"attachments":[{"name":"RecoveryPopUpSupportLink_Test_","source":"fc1a1e32-118f-4a4a-92ed-68e94bbeaa7f-attachment"}],"parameters":[],"start":1658385933370,"stop":1658385934182}